src/main/java/org/folio/orders/utils/ErrorCodes.java

```diff
@@ -0,0 +1,28 @@
+package org.folio.orders.utils;
+
+public enum ErrorCodes {
+
+  PO_NUMBER_ALREADY_EXISTS("poNumberNotUnique", "PO Number already exists"),
```

Comment:
Why stop here?  Are there other error codes that should be added here?

Reply:
We discussed it with UI guys, and agreed that the story was created to provide "poNumberNotUnique" code only at this point. Other codes will be added as necessary.